debian/control

```diff
@@ -2,12 +2,19 @@ Source: python-nodefacade
 Maintainer: Peter Brightwell <peter.brightwell@bbc.co.uk>
 Section: python
 Priority: optional
-Build-Depends: python-all (>= 2.6.6-3), debhelper (>= 7), dh-python, python-setuptools, dh-systemd, dh-apache2
+Build-Depends: python-all (>= 2.6.6-3), python3-all, debhelper (>= 7), dh-python, python-setuptools, python3-setuptools, dh-systemd, dh-apache2
 Standards-Version: 3.9.7
 
 Package: python-nodefacade
 Architecture: all
-Depends: ${misc:Depends}, ${python:Depends}, python-systemd
-Recommends: lldpd, python-nmoscommon, ips-reverseproxy-common, python-mdnsbridge
```

Comment:
For annoying reasons the joint-ri currently needs python-nmoscommon to be in recommends not depends... I'm aware this is an issue but until I have time to sort it out it needs to stay where it is please :)